commit 819ea3347e3a30a611488ceeec53650baaeb7961
Author: Samuel Thibault <samuel.thibault@ens-lyon.org>
Date:   Sun Sep 3 04:02:24 2017 +0200

    hurd: Fix rtld link
    
    dl-tunables.c now uses __access_noerrno in addition to __access.
    
            * sysdeps/mach/hurd/dl-sysdep.c (__access_noerrno): New function.

diff --git a/sysdeps/mach/hurd/dl-sysdep.c b/sysdeps/mach/hurd/dl-sysdep.c
index 6d5c1abf83..2bfa64f02a 100644
--- a/sysdeps/mach/hurd/dl-sysdep.c
+++ b/sysdeps/mach/hurd/dl-sysdep.c
@@ -557,6 +557,12 @@ __access (const char *file, int type)
   errno = ENOSYS;
   return -1;
 }
+int weak_function
+__access_noerrno (const char *file, int type)
+{
+  errno = ENOSYS;
+  return -1;
+}
 
 pid_t weak_function
 __getpid (void)
